### ISSUE TITLE
Doesn't Work on a PCjr

### ISSUE THREAD

**Issue by Fenyx4** [cambragol/advent-of-the-trinity#54]
Okay, I doubt this actually needs fixing. But thought you'd like to know. :)

**Describe the bug**
Started up my ol' [PCjr](https://en.wikipedia.org/wiki/IBM_PCjr) and when I load Trinity I just get a purple screen.

**To Reproduce**
1. Get a PCJr start the game

Lemme know if you actually want more steps after that! :)

**Expected behavior**
Not have a purple screen. ;)

**Screenshots**
https://photos.app.goo.gl/zEt4rEHH6i3KBbwb6

**Desktop (please complete the following information):**
 - OS: DOS 5.0

**Additional context**

**Comment by cambragol** [comment]
Wow! nice old hardware!

I kindof expected this. There was a point early on where I had hoped to still support playing it on a real pc. However, I hit the code limit so quick, that one of the first things I started to remove was swaths of disk drive access code etc. I removed so much of that kind of stuff. I also ripped out most of the functions for detecting graphics cards, and just hardwired the ega driver.

That being said...Title.exe should work. It was minimally modified. Seems weird that it wouldn't work either.
